plugin-server/tests/worker/ingestion/event-pipeline/runner.test.ts

```diff
@@ -454,7 +474,10 @@ describe('EventPipelineRunner', () => {
                     event: eventName,
                     team_id: 9,
                 }
-                jest.mocked(populateTeamDataStep).mockResolvedValue(event)
+                jest.mocked(populateTeamDataStep).mockResolvedValue({
+                    eventWithTeam: event,
+                    team: { id: 9, person_processing_opt_out: true } as any,
+                })
```

Comment:
suggestion: would be useful to add a case with person processing opt out set to false

Reply:
Thats the default so the other tests already cover it